docs/conf/eu/2017/news/announcing-events-volunteers.rst

```diff
@@ -0,0 +1,62 @@
+:template: 2017/eu-content.html
+
+.. post:: July 19, 2017
+   :tags: 2017, prague, events, volunteers
+
+Announcing Events and Call for Volunteers
+=========================================
+
+The conference is now just under two months away, and we hope you’re getting geared up for an exciting Write the Docs experience in beautiful Prague, Czech Republic.
+
+`Tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ are selling quickly – thanks to everyone who has already purchased theirs! We do expect the conference to sell out fully again this year, so hop off that fence you might be sitting on!
+
+If you're still debating whether to attend, check out the `Sessions <http://www.writethedocs.org/conf/eu/2017/speakers/>`_ page for the full list of presentations.
+
+Events and Activities
+---------------------
+
+As always, Write the Docs events are more than just talks. Every year we try to maximize our attendees experience and invest in the social experience of being a part of our community.
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 9th**. If you're in town early (or if you're not sure whether to come early), you're invited to join us for a 2-hour cruise on the Vltava river, with a relaxed guided tour in English, light snacks, and light drinks included. This activity is priced separately and isn't included with the conference ticket. You can find all the details in our `Boat Tour <http://www.writethedocs.org/conf/eu/2017/boat/>`_ page, or register directly on our `ticket Website <https://ti.to/writethedocs/write-the-docs-eu-2017>`_.
```

Comment:
The `(or if you're not sure whether to come early)` confuses the sentence a bit, and could probably be dropped, or made the primary part "If you're deciding if you want to come a day early"..